content/release-notes/1.16.0.md

```diff
@@ -0,0 +1,33 @@
+---
+date: 2020-06-01T08:00:00-07:00
+linktitle: "1.16.0"
+title: "1.16.0"
+kubernetes: "1.16, 1.17, and 1.18"
+---
+
+{{<features>}}
+* Support for HTTP_PROXY and NO_PROXY environment variables.
```

Comment:
Where are these environment variables supported? Is this for embedded cluster installations? 

If so: 
```suggestion
* Support for HTTP_PROXY and NO_PROXY environment variables in embedded cluster installations.
```

Reply:
unless it also works in existing cluster installs

Reply:
You can add these to the kotsadm and kotsadm-api deployments. Kurl will add them there automatically if it's configured with a proxy.